oat-sa/extension-pcisample
# Feat/adf 1734/new tooltip rendering runtime

related to https://oat-sa.atlassian.net/browse/ADF-1734

### Description

 - Updates tooltips markup to support a11y specs
 - Updates tooltips rendering behavour
 - Updates the styling of the tooltips
 - Makes tooltips rendering configurable via properties of interaction

### How to test

to be checked together with taoQtiItem counterpart https://github.com/oat-sa/extension-tao-itemqti/pull/2547

setup pci local source loading 

 - run phpfpm container `php index.php '\oat\taoQtiItem\scripts\initEventCreatorLoad'`
 - create `/config/taoQtiItem/debug_portable_element.conf.php` and set there:
 ```
<?php
    return [
        //'pciIdentifier' => 'path/to/pci/source/files',
        'textReaderInteraction' => 'pciSamples/views/js/pciCreator/ims/textReaderInteraction/'
    ];
 ```
 - reload the editor, create some tooltips and see how they are rendered
